app/controllers/submissions_controller.rb

```diff
@@ -14,6 +16,18 @@ def index
     @submissions = Submission.approved.order(created_at: :desc)
   end
 
+  def approve
```

Comment:
This method implementation isn't that great. Didn't want to spend too much time on it since it won't be used by public. The flash messages aren't even implemented so don't show on the page.

Reply:
If the flash messages aren't shown, should we skip setting them at all?

Reply:
@Aupajo I added a commit to implement flash messages 😄